demisto_sdk/commands/format/update_report.py

```diff
@@ -0,0 +1,100 @@
+import re
+from typing import Tuple
+
+from demisto_sdk.commands.common.tools import (LOG_COLORS, print_color,
+                                               print_error)
+from demisto_sdk.commands.format.format_constants import (ERROR_RETURN_CODE,
+                                                          SKIP_RETURN_CODE,
+                                                          SUCCESS_RETURN_CODE)
+from demisto_sdk.commands.format.update_generic_json import BaseUpdateJSON
+
+
+class ReportJSONFormat(BaseUpdateJSON):
+    """ReportJSONFormat class is designed to update report JSON file according to Demisto's convention.
+
+       Attributes:
+            input (str): the path to the file we are updating at the moment.
+            output (str): the desired file name to save the updated version of the YML to.
+    """
+
+    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '',
+                 no_validate: bool = False):
+        super().__init__(input, output, path, from_version, no_validate)
+
+    def run_format(self) -> int:
+        try:
+            print_color(f'\n======= Updating file: {self.source_file} =======', LOG_COLORS.WHITE)
+            self.update_json()
+            self.set_description()
+            self.set_recipients()
+            self.set_type()
+            self.set_orientation()
+            self.save_json_to_destination_file()
+            return SUCCESS_RETURN_CODE
+
+        except Exception:
+            return ERROR_RETURN_CODE
+
+    def format_file(self) -> Tuple[int, int]:
+        """Manager function for the integration YML updater."""
+        format = self.run_format()
+        return format, SKIP_RETURN_CODE
+
+    def set_type(self):
+        """
+        type is a required field for reports which is
+        limited for the following values:
+        ['pdf', 'csv', 'docx']
+        """
+        if not self.data.get('type'):
+            print_color('No type is specified for this report, would you like me to update for you? [Y/n]',
+                        LOG_COLORS.RED)
+            user_answer = input()
+            # Checks if the user input is no
+            if user_answer in ['n', 'N', 'No', 'no']:
+                print_error('Moving forward without updating type field')
+                return
+
+            print_color('Please specify the desired type: pdf | csv | docx', LOG_COLORS.YELLOW)
+            user_desired_type = input()
+            if re.match(r'(^pdf$)', user_desired_type, re.IGNORECASE):
+                self.data['type'] = 'pdf'
+            elif re.match(r'(^csv$)', user_desired_type, re.IGNORECASE):
+                self.data['type'] = 'csv'
+            elif re.match(r'(^docx$)', user_desired_type, re.IGNORECASE):
```

Comment:
why not use `user_desired_type.lower() in ('pdf', 'csv', 'docx')`?
i believe it should be more efficient than using regex